shared/lib/components.mjs

```diff
@@ -45,6 +45,9 @@ export async function load(component) {
         // Sort default to top
         ['default', ''],
 
+        // Sort urgent with non-urgent
+        ['non-', ''],
+
```

Comment:
For context, I've updated our component example sorting function to strip the text `non-`

Visually you still see "non-urgent" but [it's now sorted alongside "urgent" on the page](https://nhsuk-frontend-pr-1612.herokuapp.com/nhsuk-frontend/components/card/#non-urgent-blue)

We do similar things so sizes are sorted from S, M, L, XL etc